apps/recovery-portal/src/main/webapp/css/language-selector.css

```diff
@@ -1,6 +1,6 @@
 /*
  * <%--
- *   ~ Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *   ~ Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

Comment:
this is a new file?

Reply:
Actually yes, This was created in 2023 by me - But I added 2022 by mistaken at the time of file creation. Changed because of that